barretenberg/cpp/src/barretenberg/vm2/constraining/relations/context.test.cpp

```diff
@@ -144,6 +144,7 @@ TEST(ContextConstrainingTest, ContextSwitchingCallReturn)
 
     check_relation<context>(trace);
 
+    // TODO: Migrate to check_interaction pattern once these lookups are added in a builder
```

Comment:
Oh this is awkward. @IlyasRidhuan any reason they are not? This probably means not exercised in "prod"/bulk test.